swarm/storage/localstore_test.go

```diff
@@ -144,3 +145,67 @@ func put(store *LocalStore, n int, f func(i int64) Chunk) (hs []Address, errs []
 	}
 	return hs, errs
 }
+
+// TestGetFrequentlyAccessedChunkWontGetGarbageCollected tests that the most
+// frequently accessed chunk is not garbage collected from LDBStore, i.e.,
+// from disk when we are at the capacity and garbage collector runs. For that
+// we start putting random chunks into the DB while continuously accessing the
+// chunk we care about then check if we can still retrieve it from disk.
+func TestGetFrequentlyAccessedChunkWontGetGarbageCollected(t *testing.T) {
+	ldbCap := 10
```

Comment:
Again, I don't think we should be running tests with impossibly small db stores.

Reply:
Maybe you're right in that it doesn't matter here.

Reply:
Instead of the magic `10`. I'll put there `defaultGCRatio` as if we increment that number the test will still remain valid.

Reply:
That seems correct to me. @zelig verify?